examples/separate-config/js/every-feature-single-dest-single-ms.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Created by poggenpohlda on 19.06.2017.
+ * Separate JS file for example for better readability.
+ */
+
+$(document).ready(function() {
+    "use strict";
+    // make code pretty
+    window.prettyPrint && prettyPrint();
+
+    console.log(navigator.userAgent);
+
+    // hack for iPhone 7.0.3 multiselects bug
+    if(navigator.userAgent.match(/iPhone/i)) {
+        $('select[multiple]').each(function(){
+            /** @type {jQuery} select */
+            var select = $(this).on({
+                "focusout": function(){
+                    /** @type {string[]|number[]} values */
+                    var values = select.val() || [];
+                    setTimeout(function(){
+                        select.val(values.length ? values : ['']).change();
+                    }, 1000);
+                }
+            });
+            /** @type {string} firstOption */
+            var firstOption = '<option value="" disabled="disabled"';
+            firstOption += (select.val() || []).length > 0 ? '' : ' selected="selected"';
+            firstOption += '>Select ' + (select.attr('title') || 'Options') + '';
+            firstOption += '</option>';
+            select.prepend(firstOption);
+        });
+    }
+
+    var settings = {
+        keepRenderingFor: $.Multiselect.KeepInitialPositionFor.OPTGROUPS,
```

Comment:
This replaces the boolean keepRenderingSort. KeepInitialPositionFor is an enum where you can say "I want to store the initial position for ALL select elements or for OPTGROUPS only, or for NONE. Its a static enum object inside the Multiselect module.